app/javascript/components/MessageList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMessageList } from '../redux/messageList';
+
+const MessageList = () => {
+    const dispatch = useDispatch();
+    const message = useSelector((state) => state.updateMessageList);
+
+    useEffect(() => {
+        dispatch(fetchMessageList());
+    }, [dispatch]);
+
+    return (<h1>{message.body}</h1>)
```

Comment:
- [ ] Nice working on this component, kindly check it throwing the below error in the sreenshot.
![image](https://user-images.githubusercontent.com/48371156/193397584-d4ede82c-fde1-47ce-b0cd-085a72ef4eca.png)